packages/embark/src/lib/modules/console/index.ts

```diff
@@ -59,7 +61,13 @@ class Console {
       }
       this.events.once("console:provider:done", cb);
     });
-    this.registerEmbarkJs();
+    this.registerEmbarkJs((err?: Error |  null) => {
+      if (err) {
+        this.logger.error(err);
```

Comment:
Do we really want to say that the provider is ready when there was an error?

Reply:
If we don't call this, I think embark will hang on an error, which may or may not be better than running with an error. But it does make sense, I've added return here to prevent console ready when there is an error.